src/app/[locale]/privacy_policy/page.tsx

```
@@ -0,0 +1,247 @@
+            <h3 className="text-xl font-medium text-gray-900 mb-2">
+              Definitions
+            </h3>
+            <div className="space-y-2">
+              <p>
+                <strong>Account:</strong> means a unique account created for You
+                to access our Service or parts of our Service.
+              </p>
+              <p>
+                <strong>Affiliate:</strong> means an entity that controls, is
+                controlled by or is under common control with a party, where
+                "control" means ownership of 50% or more of the shares, equity
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)